pytket/qir/conversion/conversion.py

```diff
@@ -121,6 +121,10 @@ def __init__(
         self.cregs = _retrieve_registers(self.circuit.bits, BitRegister)
         self.target_gateset = self.module.gateset.base_gateset
 
+        self.target_gateset.add(OpType.PhasedX)
+        self.target_gateset.add(OpType.ZZPhase)
+        self.target_gateset.add(OpType.ZZMax)
```

Comment:
And `OpType.TK2`?

Reply:
Oh yes, done in @cqc-melf
[add Tk2](https://github.com/CQCL/pytket-qir/pull/46/commits/f81407e074b65916f086520b659ad91c444049cf)
[f81407e](https://github.com/CQCL/pytket-qir/pull/46/commits/f81407e074b65916f086520b659ad91c444049cf)